### DISCUSSION TITLE
Docker container instantly shuts down? [Synology DSM 7.2]

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [cross-seed/cross-seed#496] [Q&A] [answered]
I followed the guide over at www.cross-seed.org, and all of my information is correct. But whenever I turn the Docker container on in Synology's container manager, it just immediately quits and the log just says "configuration file already exists." ~~Then I find that it has wiped out my config file, setting is back to default.~~

![Screen Shot 2023-09-25 at 9 56 23 PM](https://github.com/cross-seed/cross-seed/assets/60272670/2de0d659-8848-47dd-bee0-3c98ad098516)

![Screen_Shot_2023-09-25_at_9 10 05_PM](https://github.com/cross-seed/cross-seed/assets/60272670/991fe38f-c7e7-4138-96db-eba5de715209)

Anyone know how to make this work? 

EDIT: Turns out the config.js file was never saving in the first place due to some random permissions issue (I think) that seems to be fixed now. But the container still will not run. I just set up other containers today with no problem, this is the only one I can't get to work.

**Comment by SawkeeReemo** [comment, marked as answer]
I ended up figuring this out for myself. It's a limitation with the Synology Container Manager app where it doesn't allow you to add commands to a container when it runs. So there's no way within the app to tell the cross-seed container to run with either the `daemon` or `search` command. I was able to do this in Portainer though since I don't (yet) know how to use docker-compose. 

For those trying to figure this out as well, launch Portainer, go to your container and hit the Duplicate/Edit button, then scroll down to the bottom. Click Override on the Command section and enter either `search` or `daemon` in the box there.
![Screen Shot 2023-09-25 at 10 59 50 PM](https://github.com/cross-seed/cross-seed/assets/60272670/ffa866d6-31a1-41de-b74b-8c715e96aba7)

Then set your Restart Policy to `unless stopped` and hit deploy. Then it will work!

The shortcoming in this was me. I don't know how to use Docker compose, and I spent hours trying to figure out how to before landing on this. I just ran a few tests and it works! Hope this helps someone else!

**Reply by Wyy844750180** [reply]
If possible, please provide a detailed tutorial to help us. Thank you.

**Reply by zakkarry** [reply]
> If possible, please provide a detailed tutorial to help us. Thank you.

I'd recommend you come visit us in Discord - we have several people who can help you with questions you may have, otherwise it should be relatively straight forward.

Simply provide a volume for /config and run the process `cross-seed daemon` for the container.

You can find our documentation on the container and multiple ways to run it at www.cross-seed.org
